### ISSUE TITLE
When items is not defined, optifine's implementation assumes the properties file name is the target item

### ISSUE THREAD

**Issue by boerdereinar** [SHsuperCM/CITResewn#18]
Animated texture overlays for enchanted items, e.g. sword with fire aspect, are not working for this resource pack:
https://resource-pack.com/enchantment-textures-resource-pack-1-16-3/
relevant files for fire aspect in:
assets\minecraft\optifine\cit\items\17fa\

**Comment by FrogsArePog** [comment]
The only thing that isn’t implemented in this mod is sadly enchantment item CIT’s but hopefully this will be implemented at some point in the future

**Comment by SHsuperCM** [comment]
> The only thing that isn’t implemented in this mod is sadly enchanted item CIT’s but hopefully this will be implemented at some point in the future

Yes but this issue is unrelated. The pack is using type=item with enchantment checks and it's apparently having trouble loading the mcmeta animations for the item textures..

I'll look into it(obviously separately from #15)

**Comment by SHsuperCM** [comment]
Apologies for the delay, are you sure the fire aspect sword works with optifine? the animated textures dont have any issues but the properties they use do. Some of them dont define `items` and(I guess) just assume that the properties file name is the targeted item.

If you open up the logs when loading the pack, CITResewn says if there are any issues with it, in this case:
```
[05:28:36] [Worker-Main-9/ERROR] (CITResewn) Skipped CIT: CIT must target at least one item type in Enchantment-Textures-Pack-1.16.3 -> minecraft:optifine/cit/items/11ch/trident.properties
[05:28:36] [Worker-Main-9/ERROR] (CITResewn) Skipped CIT: Cannot resolve path for model/texture in Enchantment-Textures-Pack-1.16.3 -> minecraft:optifine/cit/items/14ff/ffgoldi.png.properties
[05:28:36] [Worker-Main-9/ERROR] (CITResewn) Skipped CIT: CIT must target at least one item type in Enchantment-Textures-Pack-1.16.3 -> minecraft:optifine/cit/items/17fa/diamond_sword.properties
[05:28:36] [Worker-Main-9/ERROR] (CITResewn) Skipped CIT: CIT must target at least one item type in Enchantment-Textures-Pack-1.16.3 -> minecraft:optifine/cit/items/17fa/golden_sword.properties
[05:28:36] [Worker-Main-9/ERROR] (CITResewn) Skipped CIT: CIT must target at least one item type in Enchantment-Textures-Pack-1.16.3 -> minecraft:optifine/cit/items/17fa/iron_sword.properties
[05:28:36] [Worker-Main-9/ERROR] (CITResewn) Skipped CIT: CIT must target at least one item type in Enchantment-Textures-Pack-1.16.3 -> minecraft:optifine/cit/items/17fa/netherite_sword.properties
[05:28:36] [Worker-Main-9/ERROR] (CITResewn) Skipped CIT: CIT must target at least one item type in Enchantment-Textures-Pack-1.16.3 -> minecraft:optifine/cit/items/17fa/stone_sword.properties
[05:28:36] [Worker-Main-9/ERROR] (CITResewn) Skipped CIT: CIT must target at least one item type in Enchantment-Textures-Pack-1.16.3 -> minecraft:optifine/cit/items/17fa/wooden_sword.properties
```

I'm asking if it works with optifine because this behavior is not documented/defined anywhere and I have never seen a pack that uses this personally until now..

**Comment by boerdereinar** [comment]
It seems to work in optifine and I didn't see any exceptions in the logs.

**Comment by SHsuperCM** [comment]
Fixed in [v0.5.3](https://modrinth.com/mod/cit-resewn/version/urCxXORI)
